test/package.mjs

```diff
@@ -70,7 +70,7 @@ test('Every rule is defined in readme.md usage and list of rules in alphabetical
 	const rulesMatch = /## Rules(?<rulesText>.*?)## Deprecated Rules/ms.exec(readme);
```

Comment:
Let's use the new mark to find the table

Reply:
Updated to use the new comment markers.

Reply:
We can also use `.split('|')` to replace the regex bellow.

Reply:
I just tried switching to that but it actually turned out to be less convenient with `.split('|')`. 
- We still need a regexp to separate the url and ID
- We need to trim the whitespace around the other parts
- We need to ignore the table header and blank lines

The regexp was handling most of that for us for free...

Reply:
Okay